vercel.json

```diff
@@ -20,24 +14,20 @@
       "source": "/docs/settings/config/:path*",
       "destination": "/docs/configuration/radio/:path*"
     },
-    {
-      "source": "/docs/settings/moduleconfig",
-      "destination": "/docs/configuration/module"
-    },
     {
       "source": "/docs/settings/moduleconfig/",
       "destination": "/docs/configuration/module/"
     },
     {
       "source": "/docs/settings/moduleconfig/:path*",
-      "destination": "/docs/configuration/module/:path*"
+      "destination": "/docs/configuration/module/:path*/"
```

Comment:
this is just `module` here and not `moduleconfig` another possible issue?

Reply:
the base paths for radio and module settings were/are

      "source": "/docs/settings/config/",
      "destination": "/docs/configuration/radio/"
      
      and
      
      "source": "/docs/settings/moduleconfig/",
      "destination": "/docs/configuration/module/"